src/rpm_ostree/cli_status.rs

```diff
@@ -14,6 +15,25 @@ use std::fs;
 /// of the current state of deployments.
 const OSTREE_DEPLS_PATH: &str = "/ostree/deploy";
 
+lazy_static::lazy_static! {
+    static ref DEPLOYMENTS: IntGauge = register_int_gauge!(opts!(
+        "zincati_deployments_count",
+        "Total number of deployments found, excluding any staged deployments and the currently booted deployment."
+    )).unwrap();
+    static ref STATUS_ATTEMPTS: IntCounter = register_int_counter!(opts!(
+        "zincati_status_attempts_total",
```

Comment:
This prefix should be something like `zincati_rpm_ostree_status_cache_[...]`.

I think you may want to have two metrics here, in order to compute the cache-miss rate. You can do that by keeping a `cache_requests_total` plus a `cache_misses_total`, or a single `cache_requests_total{result=<hit|miss>}`.

This will also work as a double-check that in the steady-state we have almost zero cache misses, and all misses result in a status query and no failures.

Reply:
I was thinking we can get an approximate cache-miss rate with `cache_requests_total` and `zincati_rpm_ostree_status_attempts_total`. But yeah, there are times when `rpm_ostree_status` is called directly without even attempting to check the cache (mainly because there isn't access to the actor), so `cache_requests_total{result=<hit|miss>}` would be better :)